src/xx-cargo

```diff
@@ -79,12 +79,12 @@ if [ -n "$XX_RUSTFLAGS" ]; then
 fi
 export "CC_$(xx-info | tr - _)=$(xx-info)-clang"
 
-if which "qemu-$(RISCV64_TARGET_ARCH='' ARM_TARGET_ARCH='' xx-info march)" >/dev/null 2>&1; then
-  export "CARGO_TARGET_$(xx-info | tr '[:lower:]' '[:upper:]' | tr - _)_RUNNER=qemu-$(RISCV64_TARGET_ARCH='' ARM_TARGET_ARCH='' xx-info march)"
+if which "qemu-$(RISCV64_TARGET_ARCH='' ARM_TARGET_ARCH='' LOONG64_TARGET_ARCH='' xx-info march)" >/dev/null 2>&1; then
```

Comment:
Why these changes if there is no rust support?

Reply:
Before solving the problem of the failure of the CI, there was no adaptation of this part. Loongarch has been officially supported by Rust. I just submitted this part of the job. Let's see what the problem will be.